### ISSUE TITLE
Permissions: Assign Page Level Permission to a User

### ISSUE THREAD

**Issue by probabble** [wagtail/wagtail#4498]
It's annoying to always have to create a group to assign page level permissions to a user. 

I want to be able to assign page level permissions directly to the User object... in addition to using Groups as the primary mechanism for permissions.

**Comment by marqpdx** [comment]
Yes, this seems like a great idea. I know there is at least Django Guardian which does per object permission. Have you looked into this @probabble ?

https://github.com/django-guardian/django-guardian

It seems rather complex for the single use case of page level permission, but it might suffice.

thx,
m

**Comment by probabble** [comment]
I would like even more to be able to assign users _to a page_, giving them permissions there, and managing all of it from the context of the page that I'm assigning permissions to, not from the context of one of the thousands of users that might end up with stale permissions.

**Comment by thibaudcolas** [comment]
I’d like to see community feedback on whether this is still desirable, and if so, in what concrete scenarios. It’s annoying to create groups -- why and when exactly? With user-level permissions you would have to be very careful about who you give access to what, and I just don’t see how that scales past single-digit numbers of users.

Our internals for permissions are better equipped for this with #14398 / [RFC 102](https://github.com/wagtail/rfcs/pull/102), but I’m not clear this is desirable enough to warrant the effort on UX and overall maintenance. I also feel like this would encourage our users to lower their security standards, having to potentially carefully review the permissions of many individual user accounts.
